### PR TITLE
Feature/pelagos 4741 create list view for ips

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: assets/js/vue/components/information-product/InformationProductList.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            <DxColumn
+                type="buttons"
+                :width="60"
+            >
+                <DxButton
+                    hint="View/Edit"
+                    icon="link"
+                    :visible="true"
+                    :disabled="false"
+                    @click="viewClick"
```
Comment (fightingtexasaggie) [inline assets/js/vue/components/information-product/InformationProductList.vue:41]: I take it this is to be implemented on a different ticket?

**Review by fightingtexasaggie — APPROVED** [review]
Meets acceptance criteria on ticket - tested and reviewed

File: assets/js/vue/components/information-product/InformationProductList.vue (hunk 1 above)
Comment (mickel1138) [reply assets/js/vue/components/information-product/InformationProductList.vue:41]: Yes, that function of not done yet, different ticket.

File: assets/js/vue/components/information-product/InformationProductList.vue [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    DxHeaderFilter,
+    DxButton,
+    DxLoadPanel,
+  },
+  data() {
+    return {
+      // eslint-disable-next-line no-undef
+      data: `${Routing.generate('pelagos_api_get_all_information_product')}`,
+      researchGroups: window.researchGroups,
+      calculateFilterExpression(filterValue, selectedFilterOperation, target) {
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply assets/js/vue/components/information-product/InformationProductList.vue:114]: Did it as shown in code example, but can move it.

File: assets/js/vue/components/information-product/InformationProductList.vue [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      const text = (options.value || []).map((element) => options.column.lookup.calculateCellValue(element)).join(', ');
+      // eslint-disable-next-line no-param-reassign
+      container.textContent = text || noBreakSpace;
+      // eslint-disable-next-line no-param-reassign
+      container.title = text;
+    },
+    viewClick(e) {
+      const { id } = e.row.data;
+      window.open(`/information-product/${id}`, '_blank');
+      e.event.preventDefault();
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply assets/js/vue/components/information-product/InformationProductList.vue:137]: This how it was shown in the code example, we could try without it.

**Pull request merged**